### PR TITLE
Add remaining PEP 8 conventions

### PR DESCRIPTION
**Pull request by thejcannon**
Honestly, I thought there'd be more. But I'm happy to be wrong 😄

### CODE REVIEW

**Comment by thejcannon** [comment]
Spamming everyone, since it is a convention change. Please [approve/review the PR](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/approving-a-pull-request-with-required-reviews). (If you have no opinion, then just approve)
@davidcurtiss, @alejandro5042, @irwand, @kcuzner, @jryckman, @neilvana, @mshafer-NI, @rick-morrow,@rtzoeller, @alexweav, @sbethur

**Comment by DavidCurtiss** [comment]
👍

File: docs/Coding-Conventions.md [hunk 1]
```diff
@@ -791,6 +809,16 @@ from ministry import silly_walk
 import ministry
 ```
 
+### [O.1.6] ❌ **DO NOT** Rely on a module's imported names
+
+> 🐍 This rule stems from [PEP 8](https://www.python.org/dev/peps/pep-0008)
+
+ℹ️ Exceptions are made for:
+
+- Explcitly documented cases (E.g. `os.path`)
```
Comment (rtzoeller) [inline docs/Coding-Conventions.md:818]:
```suggestion
- Explicitly documented cases (E.g. `os.path`)
```

File: docs/Coding-Conventions.md [hunk 2]
```diff
@@ -791,6 +809,16 @@ from ministry import silly_walk
 import ministry
 ```
 
+### [O.1.6] ❌ **DO NOT** Rely on a module's imported names
```
Comment (alejandro5042) [inline docs/Coding-Conventions.md:812]: I'm not sure I understand this rule from what's written here. Perhaps expand a bit?

Comment (mshafer-NI) [reply docs/Coding-Conventions.md:812]: if module `x` imports `os`, don't import module `x` and then reference `x.os`, rather import `os` yourself.

Notice the exception, for modules that exist to bundle sub-modules (via the `__init__.py` file)

@thejcannon, maybe we should add some Avoid/Good examples?

Comment (thejcannon) [reply docs/Coding-Conventions.md:812]: I'll see if I can write one and feel good about it.

Comment (thejcannon) [reply docs/Coding-Conventions.md:812]: Added an example. 👍

**Comment by thejcannon** [comment]
I'd like to merge this soon.
@alejandro5042 @rtzoeller any more suggestions?
@alexweav, @sbethur, @stick152 Speak now or forever hold your peace.

File: docs/Coding-Conventions.md (hunk 2 above)
Comment (alejandro5042) [reply docs/Coding-Conventions.md:812]: Awesome. Thanks!

**Pull request merged**